lib/templates/updateDevice.json

```
@@ -177,7 +177,11 @@
         }
       }
     },
-    "static_attributes": {
+    "internal_attributes": {
+      "description": "Free form array of internal attributes for particular IOTAgents",
+      "type": "any"
+    },
+"static_attributes": {
```

Indentation needs to be fixed
```suggestion
    "static_attributes": {
```